### PR TITLE
Peoplefinder development fix route 53

### PR DESCRIPTION
**Pull request by digitalali-moj**
Trying to fix doubling up of domain in AWS.
image

Also moved route53 to new production namespace to allow for early domain switch

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/peoplefinder-production/resources/route53.tf [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  alias {
+    name    = "dualstack.peoplefind-elbprod-cymganexsxxq-1650047565.eu-west-1.elb.amazonaws.com."
+    zone_id = "Z32O12XQLNTSW2"
+
+    evaluate_target_health = true
+  }
+}
+
+# Production alias to the template deploy loadbalancer
+resource "aws_route53_record" "peoplefinder_route53_A_record_production" {
```
Comment (poornima-moj) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/peoplefinder-production/resources/route53.tf]:
```suggestion
resource "aws_route53_record" "peoplefinder_route53_A_record_development" {
```

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/peoplefinder-production/resources/route53.tf [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  name    = "."
```
Comment (poornima-moj) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/peoplefinder-production/resources/route53.tf]:
```suggestion
  name    = "development"
```

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/peoplefinder-production/resources/route53.tf (hunk 1 above)
Comment (digitalali-moj) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/peoplefinder-production/resources/route53.tf]: sorry, removed for now

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/peoplefinder-production/resources/route53.tf (hunk 2 above)
Comment (digitalali-moj) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/peoplefinder-production/resources/route53.tf]: sorry, removed for now

**Pull request merged**